app/communities-configs/polygon/index.js

```diff
@@ -0,0 +1,198 @@
+      padding: 12px 18px !important;
+      margin-top: .618em;
+      margin-bottom: .618em;
+
+      background-color: #7b3fe4 !important;
+      color: rgba(255,255,255,1) !important;
+      transition: background .22s ease-in-out;
+      border-radius: 300px;
+
+      :hover {
+        border-color: rgba(100,51,185,1)
+        background-color: rgba(100,51,185,1)
+      }
+    `,
+    subitems: `
+      padding: 1em;
+
+      border-top: none !important;
+      border-radius: 0 !important;
+      background: #rgba(255,255,255,1) !important;
+      transition: background .22s ease-in-out;
+      a {
+        padding: .382em;
+
+        line-height: 1.25;
+      }
+
+      @media only screen and (max-width: 991px) {
+        left: 5px;
+        top: -5px;
+
+        padding: 0 !important;
+
+        background: transparent !important;
+      }
+    `,
+  },
+  links: [
+    {
+      text: 'Scaling Solutions',
+      // href: `${domainName}dev`,
+      subitems: [
+        {
+          text: 'Scaling Solutions',
+          href: `${domainName}solutions/polygon-pos`,
+        },
+        {
+          text: 'Polygon Edge',
+          href: `${domainName}solutions/polygon-edge`,
+        },
+        {
+          text: 'Polygon Avail',
+          href: `${domainName}solutions/polygon-avail`,
+        },
+        {
+          text: 'Polygon Zero',
+          href: `${domainName}solutions/polygon-zero`,
+        },
+        {
+          text: 'Polygon Miden',
+          href: `${domainName}solutions/polygon-miden`,
+        },
+        {
+          text: 'Polygon Hermez',
+          href: `${domainName}solutions/polygon-Hermez`,
+        },
+        {
+          text: 'Polygon Nightfall',
+          href: `${domainName}solutions/polygon-nightfall`,
+        },
+      ],
+    },
+    {
+      text: 'Developers',
+      href: `${domainName}developers`,
+    },
+    {
+      text: 'Ecosystem',
+      href: `${domainName}ecosystem`,
+    },
+    {
+      text: 'Community',
+      href: `${domainName}community`,
+    },
+    {
+      text: 'Blogs',
+      href: `${domainBlog}`,
+    },
+    {
+      text: 'Get Started',
+      href: `${domainDocs}`,
+      isHighlighted: true,
+    },
+  ],
+};
+
+export const PolygonStyles = {
+  withoutCopyright: true,
+  withoutSubHeader: true,
+  poweredByPeeranha: true,
+  withoutAdditionalLinks: true,
+  signUpPageLogo: PolygonLogo,
+  favicon,
+  mobileSubHeader: (
+    <CustomMobileSubHeader config={CustomSubHeaderConfig} logo={PolygonLogo} />
+  ),
+  customSubHeader: <CustomSubHeader config={CustomSubHeaderConfig} />,
+  withoutFAQ: true,
+  fonts: {
+    h3LetterSpacing: '0.5px',
+    tagsLetterSpacing: '0.5px',
+    questionTitleLetterSpacing: '0.5px',
+    h3: 'Roboto-Regular, Arial, sans-serif',
+    main: 'Roboto-Regular, Arial, sans-serif',
+    questionItemTitle: 'Roboto-Regular, Arial, sans-serif',
+    questionTitleFont: 'Roboto-Regular, Arial, sans-serif',
+  },
+  colors: {
+    blue: '#5D6DFE',
```

Comment:
colors should be added to `style-constants`, also you can reuse some of this constants and not producing the same colors

Reply:
As far as I understand, lodash/get in style-constants.js takes color constants from array 'colors' in index.js

Reply:
You can introduce your new colors without using lodash/get functionality, it's better to control colors when they are located in one place, style-constants looks good for this.

Reply:
Wouldn't the next chain work in this case?
style-constants 'red' -> import red from style-constants in index.js -> get red from index.js in style-constants.
And yes if I use these colors later, I should specify them in style-constants, but these colors are used only for this community.

Reply:
Ok let's keep it as it is